knack/commands.py

```diff
@@ -27,6 +27,29 @@ class CLICommand(object):  # pylint:disable=too-many-instance-attributes
     def __init__(self, cli_ctx, name, handler, description=None, table_transformer=None,
                  arguments_loader=None, description_loader=None,
                  formatter_class=None, deprecate_info=None, validator=None, **kwargs):
+        """ The command object that goes into the command table.
+
+        :param cli_ctx: CLI Context
+        :type cli_ctx: knack.cli.CLI
+        :param name: The name of the command (e.g. 'mygroup mycommand')
+        :type name: str
+        :param handler: The function that will handle this command
+        :type handler: function
+        :param description: The description for the command
+        :type description: str
+        :param table_transformer: A function that transforms the command output for displaying in a table
+        :type table_transformer: function
+        :param arguments_loader: The function that defines how the arguments for the command should be loaded
+        :type arguments_loader: function
+        :param description_loader: The function that defines how the description for the command should be loaded
+        :type description_loader: function
+        :param formatter_class: The formatter for how help should be displayed
+        :type formatter_class: class
+        :param deprecate_info: Command deprecation information
```

Comment:
I would recommend a bit more detail here. I'd recommend this, assuming it is technically accurate: "Deprecation message to display when this command is invoked."

Reply:
done.